packages/eslint-plugin-pf-codemods/src/rules/v6/treeViewWarnSelectableStylingModifierRemoved/treeView-warn-selectable-styling-modifier-removed.ts

```diff
@@ -0,0 +1,32 @@
+import { Rule } from "eslint";
+import { JSXOpeningElement } from "estree-jsx";
+import { getFromPackage } from "../../helpers";
+
+// https://github.com/patternfly/patternfly-react/pull/10101
+module.exports = {
+  meta: {},
+  create: function (context: Rule.RuleContext) {
+    const { imports } = getFromPackage(context, "@patternfly/react-core");
+
+    const treeViewImport = imports.find(
+      (specifier) => specifier.imported.name === "TreeView"
+    );
+
+    return !treeViewImport
+      ? {}
+      : {
+          JSXOpeningElement(node: JSXOpeningElement) {
+            if (
+              node.name.type === "JSXIdentifier" &&
+              treeViewImport.local.name === node.name.name
+            ) {
+              context.report({
+                node,
+                message:
+                  "Selectable styling attribute (`pf-m-selectable`) has been removed in TreeView component.",
```

Comment:
Also could we just add a bit to the message here telling consumers what they might have to do because of this change/how it might impact them?

Reply:
Do you know what the message should say? Even after checking your React PR https://github.com/patternfly/patternfly-react/pull/10101 and the original issue I have no clue

Reply:
I would just let them know that they will need to update any selectors in their code or tests that depends on that classname being there, along with any snapshot tests that include the class.